lib/node_modules/@stdlib/number/float64/to-json/lib/main.js

```diff
@@ -0,0 +1,69 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var PINF = require( '@stdlib/constants/float64/pinf' );
+var NINF = require( '@stdlib/constants/float64/ninf' );
+var format = require( '@stdlib/string/format' );
+
+
+// MAIN //
+
+/**
+* Returns a JSON representation of a number.
+*
+* @param {number} x - input value
+* @throws {TypeError} must provide a number
+* @returns {Object} JSON representation
+*
+* @example
+* var json = number2json( NaN );
+* // returns { 'type': 'float64', 'value': 'NaN' }
+*/
+function number2json( x ) {
+	if ( typeof x !== 'number' ) {
+		throw new TypeError( format( 'invalid argument. Must provide a number. Value: `%d`.', x ) );
```

Comment:
```suggestion
		throw new TypeError( format( 'invalid argument. Must provide a number. Value: `%s`.', x ) );
```

I specified the wrong specifier: https://github.com/stdlib-js/stdlib/tree/3814402c0894a6379382ce657e5587738685da27/lib/node_modules/%40stdlib/string/format.